app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t("static_pages.about.title")) %>
+  <h1><%= t "static_pages.about.content_about" %></h1>
+  <p>
+    <%= t "static_pages.about.content_'about1"  %>
+    <%= link_to "http://www.railstutorial.org/" %>
```

Comment:
[IMO] This should be `<%= link_to t("static_pages.'about.content_'about2"), "http://www.railstutorial.org/" %>` or `<%= link_to "http://www.railstutorial.org/" do %>...<% end %>`